test/lotsoftypes.swagger.json

```diff
@@ -741,6 +750,12 @@
     }
   },
   "parameters": {
+    "blocknumberParam": {
+      "type": "string",
+      "description": "Used when kld-call is true, to specify the block number for the eth_call request to target. Empty or 'latest' to indicate the latest mined block. A number or a hex string to indicate a historical block height - 'x-kaleido-blocknumber' header can also be used",
```

Comment:
Think we need to do a bit of reconciliation of this text. Try and make them all short punchy lines, and defer to the docs for bigger explanation. Maybe compress the ` 'x-kaleido-blocknumber' header can also be used` on all the entries to `(header=x-kaleido-blocknumber)'